### ISSUE TITLE
Issues with 3D results for some problems

### ISSUE THREAD

**Issue by JWock82** [JWock82/PyNite#11]
My reactions appear to be incorrect. I'm adding this as a known issue to alert anyone using the program. I discovered it as I was working on a "statics check" feature. This is a high priority item I plan to resolve as soon as possible.

**Comment by JWock82** [comment]
Actually, I think this may be an issue with my statics check code, not the main program. I'll look into this and resolve this issue. For now, use caution and be advised that this may or may not be an issue.

**Comment by JWock82** [comment]
Just ran some more tests. 2D models in the XY plane don't seem to be affected by this issue. More updates to come.

**Comment by JWock82** [comment]
Update: Still working on this. It doesn't seem to affect every 3D problem. It's definitely affecting more than just reactions though. I think it may be a problem with how the 3D transformation matrix is being formulated. Tricky problem to fix.

**Comment by JWock82** [comment]
Still working on this. I found some major math errors in the textbook I was using to develop the transformation matrix. I've corrected those errors, but still seem to be getting bad results. I've got a different text arriving next week, and will check the transformation matrix against that one.

**Comment by JWock82** [comment]
I think I've got it fixed! I'm going to run some more test routines on it and make sure I'm getting results that are correct.

The text I was using had some major flaws in the transformation matrix. I ended up deriving a new transformation matrix myself. The new one is "smarter" as it keeps the local z-axis parallel to the global XZ plane. This is much more intuitive, as it assumes the global Y-axis is vertical. It is also how most singly and doubly symmetric members are placed in real world structures.

The new transformation matrix also keeps the local y-axis in the global -X direction for vertical members. This convention makes it natural to model 2D frames in the XY plane, rather than the YZ plane.
